stdlib/public/core/StringBridge.swift

```diff
@@ -177,6 +179,7 @@ extension String {
       count: length,
       elementShift: isUTF16 ? 1 : 0,
       hasCocoaBuffer: true,
+      hasNulTerminator: false,
```

Comment:
Oh I was meaning that when you call out to CF to get the pointer from _swift_stdlib_CFStringGetCStringPtr that will be null terminated. So if that returned non nil you can safely claim hasNulTerminator to true (which I would expect might make things easier to deal with later on)